examples/react-apollo-live/grafbase/schema.graphql

```diff
@@ -0,0 +1,13 @@
+schema
+  @auth(
+    providers: [{ type: oidc, issuer: "https://{{ env.CLERK_ISSUER_URL }}" }]
```

Comment:
```suggestion
    providers: [{ type: oidc, issuer: "{{ env.CLERK_ISSUER_URL }}" }]
```

We don't tend to do this in other examples or documentation. Instead we rely on using environment variables for absolute values.